From 8f222866142663d081addf168aa3b10ea42566c4 Mon Sep 17 00:00:00 2001
From: Ming Huang <ming.huang@linaro.org>
Date: Fri, 20 Jul 2018 17:57:11 +0800
Subject: [PATCH edk2-platforms v2 02/43] Silicon/Hisilicon: Separate
 PlatformArch.h

As the macro of PlatformArch.h is platform specific, so separate
PlatformArch.h to Hi1610,Hi1616 for unifying D0x.

Contributed-under: TianoCore Contribution Agreement 1.1
Signed-off-by: Ming Huang <ming.huang@linaro.org>
---
 Silicon/Hisilicon/{ => Hi1610}/Include/PlatformArch.h |  0
 Silicon/Hisilicon/Hi1616/Include/PlatformArch.h       | 35 ++++++++++++++++++++
 2 files changed, 35 insertions(+)

diff --git a/Silicon/Hisilicon/Include/PlatformArch.h b/Silicon/Hisilicon/Hi1610/Include/PlatformArch.h
similarity index 100%
rename from Silicon/Hisilicon/Include/PlatformArch.h
rename to Silicon/Hisilicon/Hi1610/Include/PlatformArch.h
diff --git a/Silicon/Hisilicon/Hi1616/Include/PlatformArch.h b/Silicon/Hisilicon/Hi1616/Include/PlatformArch.h
new file mode 100644
index 0000000000..bf0994a5e9
--- /dev/null
+++ b/Silicon/Hisilicon/Hi1616/Include/PlatformArch.h
@@ -0,0 +1,35 @@
+/** @file
+*
+*  Copyright (c) 2018, Hisilicon Limited. All rights reserved.
+*  Copyright (c) 2018, Linaro Limited. All rights reserved.
+*
+*  This program and the accompanying materials
+*  are licensed and made available under the terms and conditions of the BSD License
+*  which accompanies this distribution.  The full text of the license may be found at
+*  http://opensource.org/licenses/bsd-license.php
+*
+*  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+*
+**/
+
+
+
+#ifndef _PLATFORM_ARCH_H_
+#define _PLATFORM_ARCH_H_
+
+#define MAX_SOCKET      2
+#define MAX_DIE         4
+#define MAX_DDRC        2
+#define MAX_NODE        (MAX_SOCKET * MAX_DIE)
+#define MAX_CHANNEL     4
+#define MAX_DIMM        3
+#define MAX_RANK_CH     12
+#define MAX_RANK_DIMM   4
+// Max NUMA node number for each node type
+#define MAX_NUM_PER_TYPE 8
+
+#define S1_BASE               0x40000000000
+
+#endif
+
-- 
2.17.0

